src/Symfony/Component/Validator/Constraints/ExpressionValidator.php

```diff
@@ -56,6 +56,10 @@ private function getExpressionLanguage(): ExpressionLanguage
             $this->expressionLanguage = new ExpressionLanguage();
         }
 
+        if (false === $this->expressionLanguage->hasFunctionByName('is_valid')) {
```

Comment:
Do we really need this check?

Reply:
If we ignore the case that someone has defined his own function is_valid - we can omit this check and just rewrite it.

Reply:
let's ignore that yes, and remove the new methods on ExpressionLanguage

Reply:
Done